library/alloc/src/string.rs

```diff
@@ -1,8 +1,8 @@
-//! A UTF-8 encoded, growable string.
+//! A UTF-8–encoded, growable string.
 //!
 //! This module contains the [`String`] type, a trait for converting
-//! [`ToString`]s, and several error types that may result from working with
-//! [`String`]s.
+//! [to strings](ToString), and several error types that may result
+//! from working with [`String`]s.
```

Comment:
Maybe it's better to put the link to the trait on `a trait` instead of `to strings`, or name it explicitly:

`, [a trait](ToString) for converting to strings, ` or ``, the [`ToString`] trait for converting to strings, ``